dirsrvtests/tests/suites/logging/logging_compression_test.py

```diff
@@ -0,0 +1,119 @@
+# --- BEGIN COPYRIGHT BLOCK ---
+# Copyright (C) 2022 Red Hat, Inc.
+# All rights reserved.
+#
+# License: GPL (version 3 or any later version).
+# See LICENSE for details.
+# --- END COPYRIGHT BLOCK ---
+
+import glob
+import ldap
+import logging
+import pytest
+import os
+import time
+from lib389._constants import DEFAULT_SUFFIX
+from lib389.dseldif import DSEldif
+from lib389.topologies import topology_st as topo
+from lib389.idm.domain import Domain
+
+log = logging.getLogger(__name__)
+
```

Comment:
Could you please add `pytestmark = pytest.mark.tier1` here do the tests are run in the GitHub CI?

Reply:
> Could you please add `pytestmark = pytest.mark.tier1` here do the tests are run in the GitHub CI?

Done